src/renderer/ssf-api.ts

```diff
@@ -527,31 +524,25 @@ export class SSFApi {
     options: IScreenSharingIndicatorOptions,
     callback,
   ): void {
-    const { displayId, stream } = options;
+    const { displayId, streamId } = options;
 
-    if (!stream || !stream.active || stream.getVideoTracks().length !== 1) {
-      callback({ type: 'error', reason: 'bad stream' });
+    if (streamId && typeof streamId !== 'string') {
+      callback({ type: 'error', reason: 'bad streamId' });
       return;
     }
+
     if (displayId && typeof displayId !== 'string') {
       callback({ type: 'error', reason: 'bad displayId' });
       return;
     }
 
-    const destroy = () => {
-      throttledCloseScreenShareIndicator(stream.id);
-      stream.removeEventListener('inactive', destroy);
-    };
-
-    stream.addEventListener('inactive', destroy);
```

Comment:
https://www.electronjs.org/docs/latest/api/context-bridge#parameter--error--return-type-support - This doesn't work due to these reasons. SFE-RTC will have to be responsible for closing the screen sharing indicator (which is probably better, SymphonyElectron shouldn't do too much).